.gitignore

```diff
@@ -25,4 +25,9 @@ Gemfile.lock
 ## IntelliJ
 #################
 .idea/
-*.iml
\ No newline at end of file
+*.iml
+
+#################
+## MACOS
+#################
+.DS_STORE
```

Comment:
And use `.DS_Store`, because of there is case sensitive Mac OS X installations.

But do it in your global gitignore.